explorer/nginx/conf.d/nginx.conf.envsubst

```diff
@@ -104,7 +104,7 @@ http {
                 deny all;
             }
             # https://www.nadeau.tv/nginx-proxy_pass-dns-cache/
-            set ${DOLLAR}backend "http://prometheus:9090/prometheus/api/v1/query?query=radixdlt_core_ledger%7Bkey%3D~%22storing_per_shard%22%7D";
+            set ${DOLLAR}backend "http://prometheus:9090/prometheus/api/v1/query?query=sum%28radixdlt_core_ledger%7Bjob%3D%22nodes%22%2Ckey%3D%22storing_per_shard%22%7D%29%2A%282%5E44%2F%28sum%28radixdlt_core_shards%7Bkey%3D%22high%22%7D%29-sum%28radixdlt_core_shards%7Bkey%3D%22low%22%7D%29%29%29";
```

Comment:
Same as above. This one was a bit nastier as `2^44` is well disguised in that `%22%7D%29%2A%282%5E44%2F%28` sequence.